framework/branch-key-store.md

```diff
@@ -358,20 +441,82 @@ The operation MUST call [AWS KMS API ReEncrypt](https://docs.aws.amazon.com/kms/
 with a request constructed as follows:
 
 - `SourceEncryptionContext` MUST be the [DECRYPT_ONLY encryption context for branch keys](#decrypt_only-encryption-context).
-- `SourceKeyId` MUST be the configured `AWS KMS Key ARN` in the [AWS KMS Configuration](#aws-kms-configuration) for this keystore.
+- `SourceKeyId` MUST be the `kms-arn`.
```

Comment:
I think you need to define `kms-arn` if you want to simplify this.

Reply:
I will revert this. This might have come from rebasing after I reverted storage.

Reply:
I defined it at [`24b76b8`](https://github.com/awslabs/aws-encryption-sdk-specification/pull/297/commits/24b76b8d7b6fc956ec6fa5fe4d91c6897169d81c) commit. The definition is neutral - it could be configured (non discovery) or not configured (discovery).